modules/id5IdSystem.js

```diff
@@ -9,6 +9,42 @@ import * as utils from '../src/utils'
 import {ajax} from '../src/ajax';
 import {submodule} from '../src/hook';
 
+const ID5_ENDPOINT = 'https://id5-sync.com/g/v1';
+
+function buildUrl(config, consentData, storedUserId) {
+  const hasGdpr = (consentData && utils.isBoolean(consentData.gdprApplies) && consentData.gdprApplies) ? 1 : 0;
+  const gdprConsentString = hasGdpr ? consentData.consentString : ''
+  const urlParams = [
+    `1puid=${storedUserId ? storedUserId.id5id : ''}`,
+    `gdpr=${hasGdpr}`,
+    `gdpr_consent=${gdprConsentString}`
+  ].join('&');
+
+  return `${ID5_ENDPOINT}/${config.partner}.json?${urlParams}`;
+};
+
+function fetchId(configParams, consentData, cachedIdObj) {
+  if (!configParams || typeof configParams.partner !== 'number') {
+    utils.logError(`User ID - ID5 submodule requires partner to be defined as a number`);
+    return undefined;
```

Comment:
No need for explicit undefined. It's very common to use empty return in the js world.

Reply:
https://stackoverflow.com/questions/3330193/early-exit-from-function return alone indicates an early exit like this case.